eos-live-boot-overlayfs-setup

```diff
@@ -17,6 +17,14 @@ find_storage_partition() {
 	local root_partition=$(blkid --output device --match-token "${bootdev}")
 	[ -b "${root_partition}" ] || return 1
 
+	udevadm settle
+
+	eos-map-image-file ${root_partition} /endless/live_storage endless-live_storage
```

Comment:
It's a bikeshed but since I guess this is a disk image file, could we call it something like `/endless/persistent.img`?

Reply:
I think you are relying on this failing if the file doesn't exist but this will cause spurious error messages in the boot log for ISO images. Perhaps a quick `[[ -f $PATH ]]` could prevent that?

Reply:
I hadn't considered the log message - you are correct about my intent.

I don't think I can do the presence test quite that simply, because I don't think we mount that filesystem at all - but I could do dumpexfat and pipe the results to /dev/null to check...

I've forced pushed changes that do this.